components/multiselect/multiselect.tsx

```diff
@@ -240,6 +247,33 @@ export default class MultiSelect<T extends Value> extends React.PureComponent<Pr
         this.props.handleDelete(values);
     }
 
+    MultiValueRemove = ({children, innerProps}: any) => (
+        <div {...innerProps}>
+            {children || <CloseCircleSolidIcon className='h-4 w-4 d-flex mr-1'/>}
```

Comment:
what is the reasoning for these classNames? We wanted to migrate webapp styles to using BEM. More semantic classes would help with clarity.

Reply:
@nevyangelova These are just to avoid me creating another rule or add extra css class for the icon.
I can just add css to it, but when we just want to give basic css rules, we can just reuse the css we already have instead of adding classes for every simple thing right?